### DISCUSSION TITLE
Adding some custom components in left sidebar

### DISCUSSION THREAD

**Discussion by bhagya1101** [directus/directus#16645] [Archive]
In Directus, When I create a new Module, my left side as well as right sidebar are both empty. What should i do??
<img width="1440" alt="Screenshot 2022-11-28 at 6 24 21 PM" src="https://user-images.githubusercontent.com/117433805/204282889-5896fceb-7e3a-4a5e-9ff9-726a643b7ee8.png">

**Comment by br41nslug** [comment]
You can add components to these sidebars by mounting them in the appropriate named slots.
For the left sidebar use `#navigation` and for the right use `#sidebar`: 
```vue
<private-view>
  <template #navigation>
      <custom-navigation-component />
      <other-custom-naviation />
  </template>
  <template #sidebar>
      <custom-sidebar-component />
      <other-custom-sidebar-item />
  </template>
</private-view>
```

**Reply by bhagya1101** [reply]
Thanks a lot!!
